pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/equation/containers/SingleValueVector.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.ddp.equation.containers;
+
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+
+@EqualsAndHashCode
+@AllArgsConstructor
+public final class SingleValueVector implements AbstractVector {
+    private final BigDecimal value;
+
+    @Override
```

Comment:
My understanding is this is conforming to an interface, not overriding a super implementation. Is `@Override` required (this comment applies to `MultiValueVector` as well)?

Reply:
You're right. The annotation is not required.